website/dynamic-blocks/intro-section.js

```diff
@@ -54,11 +54,13 @@ const TableOfContents = () => {
 		const headings = [];
 		if (containerRef && containerRef.current) {
 			const container = containerRef.current.closest('.slate-container') || document;
-			container.querySelectorAll('[data-toc]').forEach((h, i) => {
-				headings[i] = <TableLink key={`nav-${i}`} headingId={h.id} headingText={h.innerText} />;
-			});
+			setTimeout(() => {
```

Comment:
@dominikwilkowski The page-content list wasn't being shown because the headings are rendered after this has run so it can't find them. Not sure of the best way to solve this.